osl/utils/create_neuromag306_info.py

```diff
@@ -0,0 +1,100 @@
+                              'MEG0312', 'MEG0323', 'MEG0332', 'MEG0342', 'MEG0412', 'MEG0423', 'MEG0433', 'MEG0442',
+                              'MEG0512', 'MEG0522', 'MEG0533', 'MEG0543', 'MEG0612', 'MEG0623', 'MEG0632', 'MEG0643',
+                              'MEG0712', 'MEG0722', 'MEG0732', 'MEG0742', 'MEG0812', 'MEG0823', 'MEG0912', 'MEG0922',
+                              'MEG0933', 'MEG0943', 'MEG1012', 'MEG1022', 'MEG1033', 'MEG1042', 'MEG1113', 'MEG1122',
+                              'MEG1132', 'MEG1143', 'MEG1212', 'MEG1222', 'MEG1233', 'MEG1242', 'MEG1313', 'MEG1322',
+                              'MEG1332', 'MEG1343', 'MEG1413', 'MEG1422', 'MEG1432', 'MEG1443', 'MEG1513', 'MEG1523',
+                              'MEG1532', 'MEG1542', 'MEG1612', 'MEG1623', 'MEG1633', 'MEG1642', 'MEG1712', 'MEG1723',
+                              'MEG1733', 'MEG1742', 'MEG1812', 'MEG1823', 'MEG1833', 'MEG1842', 'MEG1913', 'MEG1922',
+                              'MEG1933', 'MEG1942', 'MEG2012', 'MEG2022', 'MEG2033', 'MEG2043', 'MEG2112', 'MEG2123',
+                              'MEG2132', 'MEG2142', 'MEG2213', 'MEG2222', 'MEG2232', 'MEG2243', 'MEG2313', 'MEG2322',
+                              'MEG2333', 'MEG2342', 'MEG2413', 'MEG2422', 'MEG2432', 'MEG2443', 'MEG2513', 'MEG2523',
+                              'MEG2532', 'MEG2542', 'MEG2613', 'MEG2622', 'MEG2632', 'MEG2643']
+
+channels['mag'] = ['MEG0111', 'MEG0121', 'MEG0131', 'MEG0141', 'MEG0211', 'MEG0221', 'MEG0231', 'MEG0241', 'MEG0311',
+                   'MEG0321', 'MEG0331', 'MEG0341', 'MEG0411', 'MEG0421', 'MEG0431', 'MEG0441', 'MEG0511', 'MEG0521',
+                   'MEG0531', 'MEG0541', 'MEG0611', 'MEG0621', 'MEG0631', 'MEG0641', 'MEG0711', 'MEG0721', 'MEG0731',
+                   'MEG0741', 'MEG0811', 'MEG0821', 'MEG0911', 'MEG0921', 'MEG0931', 'MEG0941', 'MEG1011', 'MEG1021',
+                   'MEG1031', 'MEG1041', 'MEG1111', 'MEG1121', 'MEG1131', 'MEG1141', 'MEG1211', 'MEG1221', 'MEG1231',
+                   'MEG1241', 'MEG1311', 'MEG1321', 'MEG1331', 'MEG1341', 'MEG1411', 'MEG1421', 'MEG1431', 'MEG1441',
+                   'MEG1511', 'MEG1521', 'MEG1531', 'MEG1541', 'MEG1611', 'MEG1621', 'MEG1631', 'MEG1641', 'MEG1711',
+                   'MEG1721', 'MEG1731', 'MEG1741', 'MEG1811', 'MEG1821', 'MEG1831', 'MEG1841', 'MEG1911', 'MEG1921',
+                   'MEG1931', 'MEG1941', 'MEG2011', 'MEG2021', 'MEG2031', 'MEG2041', 'MEG2111', 'MEG2121', 'MEG2131',
+                   'MEG2141', 'MEG2211', 'MEG2221', 'MEG2231', 'MEG2241', 'MEG2311', 'MEG2321', 'MEG2331', 'MEG2341',
+                   'MEG2411', 'MEG2421', 'MEG2431', 'MEG2441', 'MEG2511', 'MEG2521', 'MEG2531', 'MEG2541', 'MEG2611',
+                   'MEG2621', 'MEG2631', 'MEG2641']
+
+for i in channels.keys():
+    channels[i] = sorted(channels[i])
+
+with open('utils/neuromag306_info.yml', 'w') as outfile:
```

Comment:
if the py file and yml file are in the same directory, then i think the following should be fine:
`str(Path(__file__).parent) + "neuromag306_info.yml"`